px-vis-line-canvas.html

```diff
@@ -523,6 +523,10 @@
           }
         }
 
+        var dashPattern = _this.completeSeriesConfig[_this.seriesId] && _this.completeSeriesConfig[_this.seriesId]["dashPattern"] ? _this.completeSeriesConfig[_this.seriesId]["dashPattern"] : '';
+        dashPattern = eval('[' +dashPattern +']');
```

Comment:
Is this to parse the string? Use JSON.parse instead? Can svg dash pattern accept an array? Something to look into?